weatherv3/www/js/controllers.js

```diff
@@ -0,0 +1,116 @@
+    console.log('inside home');
+    $scope.city  = DataStore.city;
+    var latitude  =  DataStore.latitude;
+    var longitude = DataStore.longitude;
+
+    //some code found here: https://www.binpress.com/tutorial/a-simple-weather-forecast-app-with-ionic-framework-and-forecastio/107
+    Weather.getCurrentWeather(latitude,longitude).then(function(resp) {
+      var current = resp.data;
+      $scope.temperature = Math.round(current.currently.temperature) + "°F";
+      $scope.apparentTemperature = Math.round(current.currently.apparentTemperature) + "°F";
+      $scope.summary = current.currently.summary.toLowerCase();
+      $scope.tempDif = Math.abs(Math.round(current.currently.temperature) - Math.round(current.currently.apparentTemperature));
+      $scope.windSpeed = Math.round(current.currently.windSpeed);
+      var intensity = current.currently.precipIntensity;
+
+      $scope.precipIntensity = 0;
+      if (intensity == 0) {
+        $scope.precipIntensity = 0;
+      } else if (intensity < .002) {
+        $scope.precipIntensity = "very lightly";
+      } else if (intensity < .017) {
+        $scope.precipIntensity = "lightly";
+      } else if (intensity < .1) {
+        $scope.precipIntensity = "moderately hard";
+      } else {
+        $scope.precipIntensity = heavily;
+      }
+      $scope.precipProbability = current.currently.precipProbability;
+      $scope.precipType = current.currently.precipType;
+      $scope.nearestStormDistance = current.currently.nearestStormDistance;
+
+      console.log('tempDif: ', $scope.tempDif);
+      console.log('GOT CURRENT', current);
+      //debugger;
+    }, function(error) {
+      alert('Unable to get current conditions');
+      console.error(error);
+    });
+
+})
+
+.controller('weekCtrl', function($scope,$state,$filter,Weather,DataStore,WeekDays) {
+    //read default settings into scope
+    console.log('inside weekCtrl');
+    $scope.city  = DataStore.city;
+    var latitude  =  DataStore.latitude;
+    var longitude = DataStore.longitude;
+
+    //call getCurrentWeather method in factory ‘Weather’
+    Weather.getCurrentWeather(latitude,longitude).then(function(resp) {
+      $scope.current = resp.data;
+      console.log('GOT CURRENT', $scope.current);
+      theForecast();
+      //debugger;
+    }, function(error) {
+      alert('Unable to get current conditions');
+      console.error(error);
+    });
+
+    var weekDays = WeekDays;
+    var today = WeekDays.getToday();
+    //returns an array with info for the next 7 days
+    var theForecast = function() {
+      console.log('in theForecast function. value of current: ', $scope.current);
+      var d = new Date();
+      result = new Array(7);
+      var n = today;
+
+      $scope.barTypes = ['bar-calm', 'bar-balanced', 'bar-energized', 'bar-assertive', 'bar-royal', 'bar-dark','bar-positive', 'bar-calm', 'bar-balanced', 'bar-energized', 'bar-assertive', 'bar-royal', 'bar-dark','bar-positive'];
+      $scope.barFirst = n;
```

Comment:
I had implemented it in the html using a for loop, but I don't think I'll be using this anymore. For the next version I'll probably use a similar background for all three tabs.